server/http.go

```diff
@@ -192,6 +178,14 @@ func (p *Plugin) completeUserOAuth(w http.ResponseWriter, r *http.Request) {
 </html>
 `
 
+	post := &model.Post{
+		UserId:    p.botUserID,
+		ChannelId: channelID,
+		Message:   "User successfully connected to MS Teams Meetings.",
```

Comment:
This is just being send to the user, so referring to them as "user" doesn't look right to me.  Perhaps "You have successfully connected to MS Teams Meetings."

Reply:
@wiggin77 
Updated the message to `You have successfully connected to MS Teams Meetings.`
Please have a look. Currently, the disconnect message is `User disconnected from MS Teams Meetings.`
Do you think we should update it too?